types/koa/test/default.ts

```diff
@@ -40,7 +40,9 @@ app.use((ctx: Koa.Context, next) => {
         const ms = end - start;
         ctx.logger.info(`${ctx.method} ${ctx.url} - ${ms}ms`);
         ctx.user = {}; // undeclared property
-        ctx.assert(true, 404, 'Yep!');
+        ctx[Symbol('locale')] = 'en-US'; // symbol property key
```

Comment:
Can you add `$ExpectType` expecations, for example:

```
ctx[Symbol('locale')] = 'en-US';
ctx[Symbol('locale')]; // $ExpectType string
```

Reply:
Can you also add `string` example: `ctx['custom'] = 123`

Reply:
`ctx[Symbol('locale')]`'s `$ExpectType` is always `any`:

```log
ERROR: 44:9  expect  TypeScript@5.2 expected type to be:
  string
got:
  any
```

Reply:
This is the issue with `any` as a type. Can you try changing the signature to `[key: PropertyKey]: unknown;`?

Reply:
Please don’t change `any` to `unknown` in this PR. That would be a significant breaking change for an extremely heavily relied upon package, and we could not approve it in a patch release like this.

Reply:
`[key: PropertyKey]: unknown;` does not work. Try to assert `ctx[Symbol('locale')]; // $ExpectType string`:
```log
ERROR: 45:9  expect  TypeScript@5.2 expected type to be:
  string
got:
  unknown
```

Reply:
Lets keep `any` and expect the same in the test.